### PR TITLE
feat(links+numbers): add links tag and add icon to numbers tag 

### PR DESCRIPTION
**Pull request by verarojman**
Screen Shot 2022-02-17 at 16 38 18

image

- [x] Add link and document icons
- [x] Add numbers description icon
- [x] Style numbers in rows

### CODE REVIEW

**Review by agustibr — APPROVED** [review]
👏😃

File: _includes/svg/icon-quotes.liquid [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" class="svg-icon-quotes {{ include.class }}" alt="{{ include.alt }}">
```
Comment (agustibr) [inline _includes/svg/icon-quotes.liquid:1]: 🤔 is this svg `icon-quotes`(in plural) different to `icon-quote` (in singular)? 
If it's duplicate we should remove on of them 😄

Comment (verarojman) [reply _includes/svg/icon-quotes.liquid:1]: 😬 it's a duplicate, yes

Comment (agustibr) [reply _includes/svg/icon-quotes.liquid:1]: Ok, no problem!

Thanks

**Pull request merged**